package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
```

Comment:
- [ ] It would be nice if you add Future Feature section in your readme file to make your readme descriptive and professional :blush: